Dockerfile

```diff
@@ -32,7 +32,11 @@ RUN pnpm run build
 # The second stage builds the backend.
 FROM python:3.11-bullseye as backend-builder
 WORKDIR /phoenix
-COPY ./ /phoenix/
```

Comment:
copying too much on local build (e.g. node_modules)

Reply:
Node modules shouldn't be copied, they are in `.dockerignore`. I think it keeps the Dockerfile simpler to avoid copying individual files. I'd also mention that none of the contents of this stage are included in the final build except for what is explicitly copied in the final step.

Reply:
ah ok. i guess it's copying the tox folder

Reply:
Got it. Can we add to `.dockerignore` instead?

Reply:
There are also other things like packages and docs. It's probably easier to copy just the specific files.